### PR TITLE
SBERDOMA-490 Added citizens overview page

### PR DESCRIPTION
**Pull request by SavelevMatthew**
Citizens are now separated with Users (Employees)
Added citizens overview page

### CODE REVIEW

File: apps/condo/domains/contact/hooks/useTableColumns.tsx [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+    clearFilters: () => void
+    showClearButton?: boolean
+}
+
+const FilterContainer: React.FC<IFilterContainerProps> = (props) => {
+    const intl = useIntl()
+    const ResetLabel = intl.formatMessage({ id: 'filters.Reset' })
+
+    return (
+        <div style={{ padding: 16 }}>
```
Comment (toplenboren) [inline apps/condo/domains/contact/hooks/useTableColumns.tsx:23]: I think its better to use emotion's css property when dealing with native browser elements. Inserting styles is more expensive during render time then inserting css class

We can rewrite this code like so:

```
import { css } from 'emotion'

const FilterContainerDiv = css`padding: 16`
```

File: apps/condo/domains/contact/utils/helpers.ts [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+
+        try {
+            if (CITIZEN_TABLE_COLUMNS.includes(columnKey) && SORT_ORDERS[sortKey]) {
+                return {
+                    columnKey,
+                    order: SORT_ORDERS[sortKey],
+                }
+            }
+        } catch (e) {
+            // TODO(Dimitreee): send error to sentry
```
Comment (toplenboren) [inline apps/condo/domains/contact/utils/helpers.ts:107]: 👀

File: apps/condo/domains/contact/utils/helpers.ts [hunk 3]
```diff
@@ -0,0 +1,148 @@
+import { SortOrder } from 'antd/es/table/interface'
```
Comment (dkoviazin) [inline apps/condo/domains/contact/utils/helpers.ts:1]: Add tests for utils, like 
```bash
/domains/ticket/utils/helpers.spec.ts
```

**Comment by pahaz** [comment]
We don't have citizen in our architect vocabulary %) It may confuse

**Comment by pahaz** [comment]
It's better to use a `contact` word inside the code.

File: apps/condo/domains/contact/utils/helpers.ts (hunk 3 above)
Comment (SavelevMatthew) [reply apps/condo/domains/contact/utils/helpers.ts:1]: in progress :)

**Comment by SavelevMatthew** [comment]
> It's better to use a `contact` word inside the code.

Translation fixed 👍

File: apps/condo/domains/contact/hooks/useTableColumns.tsx (hunk 1 above)
Comment (SavelevMatthew) [reply apps/condo/domains/contact/hooks/useTableColumns.tsx]: done 👍

File: apps/condo/domains/contact/utils/helpers.ts (hunk 3 above)
Comment (SavelevMatthew) [reply apps/condo/domains/contact/utils/helpers.ts:1]: done

**Pull request closed without merging**